### PR TITLE
feat : Authentication

### PR DESCRIPTION
**Pull request by anuj-kumary**
This PR includes:-

- Sign in Page With Functionality
- Sign up Page With Functionality

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <>
+      <Typography
+        sx={{ width: '100%', textAlign: 'center' }}
+        component='h1'
+        variant='h2'
+      >
+        Welcome to Mitro
+      </Typography>
+    </>
```
Comment (romabulani) [inline src/pages/Home/Home.jsx:13]: Here you can remove the fragment

**Review by romabulani — APPROVED** [review]
- Liked that you have used Material UI
- Clean and modular code
- Suggestion : You should keep username instead of email id. In the current code, people with different email IDs can use same username

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/pages/Home/Home.jsx:13]: Done

**Comment by anuj-kumary** [comment]
> * Liked that you have used Material UI
> * Clean and modular code
> * Suggestion: You should keep username instead of email id. In the current code, people with different email IDs can use same username

Heyy  @romabulani  Thanks for the Suggestion I made changes just check once

**Comment by romabulani** [comment]
> > * Liked that you have used Material UI
> > * Clean and modular code
> > * Suggestion: You should keep username instead of email id. In the current code, people with different email IDs can use same username
> 
> Heyy @romabulani Thanks for the Suggestion I made changes just check once

Yess now it looks good to merge.

**Review by varnita619 — APPROVED** [review]
Code is clean
UI looks good
Ready to merge

**Pull request merged**